benchmarks/parsing.cpp

```diff
@@ -1,18 +1,20 @@
 #include <iostream>
 #include <chrono>
 #include <symengine/parser.h>
+#include <symengine/parser/parser.h>
 
 using SymEngine::Basic;
 using SymEngine::RCP;
 using SymEngine::print_stack_on_segfault;
-using SymEngine::parse;
+using SymEngine::Parser;
 using SymEngine::parse_old;
 
 int main(int argc, char *argv[])
 {
     SymEngine::print_stack_on_segfault();
 
     RCP<const Basic> a;
+    Parser p;
```

Comment:
Can you revert this change? I'll send a PR to fix the slowness because of initialization. (https://github.com/isuruf/symengine/tree/use_bison)

Reply:
Why don't you revert the parsing.cpp once you send the PR? I don't know exactly how you intend to fix it, so I was hoping getting this merged, and then we can benchmark your new PR against this, and if the performance stays the same, we can merge yours also.

If I revert it, then we will never know easily that your PR actually fixes the performance fully.

Reply:
Sure.